FSJ_django20_project/FSJ/urls.py

```
@@ -1 +1 @@
-from django.contrib.auth import views as auth_views
from django.urls import path
from . import views

urlpatterns = [
    path('', views.redirect_to_home, name='redirect_to_home'),
    path('login/', auth_views.LoginView.as_view(template_name="FSJ/login.html", redirect_authenticated_user=True), name='login'),
    path('logout/', auth_views.LogoutView.as_view(), name='logout'),
    path('home/', views.home, name='home'),
    path('profile/', views.profile, name='profile'),
]
\ No newline at end of file
+from django.contrib.auth import views as auth_views
from django.urls import path
from . import views

urlpatterns = [
    path('', views.redirect_to_home, name='redirect_to_home'),
    path('login/', auth_views.LoginView.as_view(template_name="FSJ/login.html", redirect_authenticated_user=True), name='login'),
    path('logout/', auth_views.LogoutView.as_view(), name='logout'),
    path('home/', views.home, name='home'),
    path('profile/', views.profile, name='profile'),
    path('student/', views.coordinator_studentlist, name = 'student'),
    path('student/<str:usr_ccid>/', views.coordinator_studentdetail, name = 'studentdetail')
]

```

Logout button from /student/ results in a HTTP404 being raised instead of logging the user out of the application.